codice/ddf
# DDF-1366 admin-poller-service tech debt

#### What does this PR do?
This PR
- Updates log levels in AdminPollerServiceBean to follow DDF guidelines: https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels
- Updates the name of `admin-poller-service` to be consistent
Before:
![image](https://user-images.githubusercontent.com/8041246/46050310-45281300-c0e8-11e8-9d66-fe0491e8edff.png)
After:
![image](https://user-images.githubusercontent.com/8041246/46050314-48230380-c0e8-11e8-939f-e3e7f8e8ecc7.png)
#### Who is reviewing it? 
@kcover @peterhuffer @pvargas 
#### Ask 2 committers to review/merge the PR and tag them here.
@clockard
@vinamartin
#### How should this be tested?
- CI build
- Confirm that the catalog-admin-poller-service is installed with the catalog-app.
#### What are the relevant tickets?
I noticed this tech debt while working [DDF-1366](https://codice.atlassian.net/browse/DDF-1366).
#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
